libobs/obs-internal.h

```diff
@@ -1205,6 +1208,9 @@ struct obs_encoder {
 
 	size_t mixer_idx;
 
+	bool gpu_scaling_enabled;
+	enum obs_scale_type scale_type;
+
```

Comment:
Do we need both an `enabled` flag and a `scale_type`? Wouldn't `OBS_SCALE_DISABLE` imply scaling is disabled?

Reply:
I can't think of a good reason to keep both if we want `OBS_SCALE_DISABLE` to imply scaling is disabled; not sure when I'll get around to update this, probably on Monday

Reply:
@notr1ch: I changed this in <https://github.com/obsproject/obs-studio/pull/9043/commits/cbd606da0354af62b98cfaee7fb4836ed11e8c44>; at that point the name `obs_encoder_enable_gpu_scaling` didn't seem to fit super well anymore, let me know what you think

Reply:
Looks good to me, I had the same thought but wanted to keep it to one discussion.